examples/ftd_api_scripts/timerange-access-rule-association.py

```diff
@@ -0,0 +1,154 @@
+        return
+    access_rule = {
+        "name": "Test_Python_Request_For_Access_Rule_5",
+        "sourceZones": [],
+        "destinationZones": [],
+        "sourceNetworks": [],
+        "destinationNetworks": [],
+        "sourcePorts": [],
+        "destinationPorts": [],
+        "timeRangeObjects": [time_range],
+        "ruleAction": "TRUST",
+        "eventLogAction": "LOG_BOTH",
+        "identitySources": [],
+        "users": [],
+        "logFiles": "false",
+        "destinationDynamicObjects": [],
+        "sourceDynamicObjects": [],
+        "type": "accessrule"
+    }
+    access_rule = post_access_rule(host, port, access_token, access_rule, parent_policy_id)
+    if not access_rule:
+        print('Unable to create access rule')
+        return
+    access_rule_update = {
+        "version": access_rule['version'],
+        "name": "Test_Python_Request_For_Access_Rule__5_Updated",
+        "ruleId": access_rule['ruleId'],
+        "sourceZones": [],
+        "destinationZones": [],
+        "sourceNetworks": [],
+        "destinationNetworks": [],
+        "sourcePorts": [],
+        "destinationPorts": [],
+        "timeRangeObjects": [time_range],
+        "ruleAction": "PERMIT",
+        "eventLogAction": "LOG_BOTH",
+        "identitySources": [],
+        "users": [],
+        "logFiles": "false",
+        "destinationDynamicObjects": [],
+        "sourceDynamicObjects": [],
+        "type": "accessrule"
+    }
+    updated_access_rule = update_access_rule(host, port, access_token, access_rule['id'], access_rule_update, parent_policy_id)
+    if not updated_access_rule:
+        print('Unable to update access rule')
+        return
+    access_rule = get_access_rule(host, port, access_token, updated_access_rule['id'], parent_policy_id)
+    if not access_rule:
+        print('Unable to get access rule')
+        return
+    print('Access rule name is {}'.format(access_rule['name']))
+    access_rule_update = {
+        "version": access_rule['version'],
+        "name": "Test_Python_Request_For_Access_Rule_Deleted_TRO_5",
+        "ruleId": access_rule['ruleId'],
+        "sourceZones": [],
+        "destinationZones": [],
+        "sourceNetworks": [],
+        "destinationNetworks": [],
+        "sourcePorts": [],
+        "destinationPorts": [],
+        "ruleAction": "PERMIT",
+        "eventLogAction": "LOG_BOTH",
+        "identitySources": [],
+        "users": [],
+        "logFiles": "false",
+        "destinationDynamicObjects": [],
+        "sourceDynamicObjects": [],
+        "type": "accessrule"
+    }
+    updated_access_rule = update_access_rule(host, port, access_token,
+                                                             access_rule['id'], access_rule_update, parent_policy_id)
+    if not updated_access_rule:
+        print('Unable to update access rule')
+        return
+    print('Access rule name is {}'.format(updated_access_rule['name']))
+
```

Comment:
Return True if successful, otherwise False

Reply:
Done, addressed in https://github.com/stleary-cisco/ftd-rest-api-cookbook/pull/7